Source/DafnyCore/Options/DafnyCommands.cs

```diff
@@ -49,7 +49,9 @@ public static class DafnyCommands {
     CommonOptionBag.VerificationCoverageReport,
     CommonOptionBag.ExtractCounterexample,
     CommonOptionBag.ShowInference,
-    CommonOptionBag.ManualTriggerOption
+    CommonOptionBag.ManualTriggerOption,
+    CommonOptionBag.FilterSymbol,
```

Comment:
I think `VerificationOptions` is too wide, because it includes everything that can do verification, such as `dafny build` and `dafny translate`. Those commands have no business filtering verification.

Reply:
I think that's reasonable. I'll merge it as-is with your update.